containers/systemd-service

```diff
@@ -59,13 +57,22 @@ function run {
   fi
 }
 
+function sdproxy() {
+  SD_NOTIFY_PROXY=/usr/local/pf/bin/sdnotify-proxy
+  SD_SOCK=/tmp/$1.sock
+  CR_SOCK=/var/run/$1.sock
+  WATCHDOG_USEC_D=10000000
+  echo "SDPROXY cli: $SD_NOTIFY_PROXY $SD_SOCK /usr/bin/docker run --volume=$SD_SOCK:$CR_SOCK --env NOTIFY_SOCKET=$CR_SOCK --env WATCHDOG_USEC=$WATCHDOG_USEC_D ${@:2}"
+  $SD_NOTIFY_PROXY $CR_SOCK /usr/bin/docker run --volume=$CR_SOCK:$CR_SOCK --env NOTIFY_SOCKET=$CR_SOCK --env WATCHDOG_USEC=$WATCHDOG_USEC_D ${@:2}
+}
+
 function run_img {
   img="$1"
   name="$2"
   args="$3"
   cmd="$4"
   cd /usr/local/pf/
-  docker container rm -f $name
-  echo "Running with args $args"
-  docker run $args --name=$name $img $cmd
+  docker container rm -f $name 1&2> /dev/null
+  #echo "Running with args $args"
```

Comment:
Keeping this line here can be useful for troubleshooting

Reply:
It has been replaced by "SDPROXY cli" but as you mentioned it could be out of sync with the command line.
I will bring it back.

Reply:
Done